tsconfig.6jyxozowiar.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
What is this file needed for?

Reply:
It's an artifact of `tsc-files` when running through `lint-staged`. Can be removed.